console-subscriber/src/aggregator/mod.rs

```diff
@@ -220,9 +220,11 @@ impl Aggregator {
             // to be woken when the flush interval has elapsed, or when the
             // channel is almost full.
             let mut drained = false;
+            let mut counts = EventCounts::new();
             while let Some(event) = recv_now_or_never(&mut self.events) {
                 match event {
                     Some(event) => {
+                        counts.update(&event);
```

Comment:
take it or leave it: if we're concerned about the performance cost of aggregating these counts if we're not logging the `DEBUG` event, we could write something like this:
```suggestion
                        if tracing::event_enabled!(tracing::Level::DEBUG) {
                            counts.update(&event);
                        }
```

Reply:
but, i'm not sure if this is actually more efficient than just doing a single add per event. so i don't know if i would bother...

Reply:
Given the updates are probably going to be very fast, I think I'll just sum them up anyway.